### PR TITLE
Try to parse message as JSON text

### PR DESCRIPTION
**Pull request by ykitamura-mdsol**
Trying to log as JSON text

```python
def lambda_handler(event, context):
    print(json.dumps({"hello": "lambda"}))
```
⬇️ 

**Before**
Screen Shot 2021-02-12 at 6 31 17 PM

**After**
Screen Shot 2021-02-12 at 6 44 22 PM

Also, added customizations to bake the SumoLogic URL in the code and to skip deploying layer

@mdsol/architecture-enablement @jcarres-mdsol

### CODE REVIEW

File: lambda-extensions/sumoclient/sumoclient.go [hunk 1]
```diff
@@ -179,6 +179,7 @@ func (s *sumoLogicClient) getLogStream() string {
 
 func (s *sumoLogicClient) enhanceLogs(msg responseBody) {
 	s.logger.Debugln("Enhancing logs")
+	var jsonMessage interface{}
 	for _, item := range msg {
 		// item["FunctionName"] = s.config.FunctionName
```
Comment (jcarres-mdsol) [inline lambda-extensions/sumoclient/sumoclient.go:184]: Why keep this? We could overwrite the whole thing and make logs coming from here to look like normal logs

File: lambda-extensions/config/config.go [hunk 2]
```diff
@@ -41,9 +41,13 @@ var validLogTypes = []string{"platform", "function", "extension"}
 
 // GetConfig to get config instance
 func GetConfig() (*LambdaExtensionConfig, error) {
+	sumoHttpEndpoint, ret := os.LookupEnv("SUMO_HTTP_ENDPOINT")
```
Comment (jcarres-mdsol) [inline lambda-extensions/config/config.go:44]: This is just for logs right? Or do traces also go through this?

Comment (ykitamura-mdsol) [reply lambda-extensions/config/config.go:44]: Yes, just for logs.

File: lambda-extensions/sumoclient/sumoclient.go (hunk 1 above)
Comment (ykitamura-mdsol) [reply lambda-extensions/sumoclient/sumoclient.go:184]: okay, let me try to simplify it.

Comment (ykitamura-mdsol) [reply lambda-extensions/sumoclient/sumoclient.go]: updated 4d7cd56

**Pull request merged**